windows/src/engine/keyman/main.pas

```diff
@@ -181,6 +181,10 @@ procedure InitialiseRegistrySecurity;
     if r.OpenKey(SRegKey_KeymanRoot_CU, True) then
```

Comment:
Would this be the same?
```
if r.OpenKey(SRegKey_KeymanRoot_CU, True) or r.OpenKey('\' + SRegKey_KeymanEngineRoot_CU, True) then
```

Reply:
No, because I am applying the permission to each key in turn. That change would only allow the permission to be applied to the `SRegKey_KeymanEngineRoot_CU` key.